pyregion/wcs_helper.py

```diff
@@ -1,359 +1,91 @@
-import re
 import numpy as np
-from astropy.extern import six
+from astropy.coordinates import SkyCoord
 from astropy.wcs import WCS
-from astropy.io.fits import Header
-from .extern.kapteyn_celestial import skymatrix, longlat2xyz, dotrans, xyz2longlat
-from .extern import kapteyn_celestial
+from astropy.wcs.utils import proj_plane_pixel_scales
```

Comment:
Was `proj_plane_pixel_scales` added in Astropy 1.1 or 1.2?

Maybe we should start to be explicit and put `astropy>=1.1` (or whatever we really support and test) here?
https://github.com/astropy/regions/blob/master/setup.py#L109